runtime/syntax/bat.yaml

```diff
@@ -2,7 +2,7 @@ filetype: batch
 
 detect:
   filename: "(\\.bat$|\\.cmd$)"
-  # signature: ""
+  # header: ""
```

Comment:
I suggest to remove this deactivated and thus unneeded line. If it's needed the line must be touched anyway.